ngraph_bridge/ngraph_capture_variables.cc

```diff
@@ -201,10 +201,40 @@ Status CaptureVariables(Graph* graph, const std::set<string> skip_these_nodes) {
         }
 
         replaced_nodes.push_back(node);
-      } else if (node->type_string() == "PrefetchDataset") {
-        // Collect the prefetch_node so that we can add
-        // the NGraphWriteToDevice Op after this one
-        prefetch_node = node;
+      } else if (node->type_string() == "MakeIterator") {
+        if (std::getenv(NGraphPrefetchSharedResouce::NGRAPH_TF_USE_PREFETCH) !=
```

Comment:
Please add this to --var build also.

Reply:
We can think of another way to do it, so that it is easily accessible for both var and non var